platform/tests/stacks/up/up_test.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+amp="amp -s localhost"
+
+$amp login --name user1 --password password
+$amp stack up -c platform/stacks/visualizer.stack.yml
```

Comment:
I'd prefer to see an absolute path, you're making an assumption that the test is run at the root of the project